File: 2_POO vs Estruturada_alunos/Python/poo.py
#Classe Cachorro (É como se fosse um molde de criação)
class Cachorro:
    def __init__(self, nome, comida, sono):
        self.nome = nome
        self.comida = comida
        self.sono = sono
    
    def comer(self):
        self.comida -= 1
    
    def dormir(self):
        self.sono = False

# Criando (Objetos)
cachorro_1 = Cachorro("Nelson", 3, False)
cachorro_2 = Cachorro("Jeremias", 1, True)

# Usando métodos
cachorro_1.comer()
cachorro_2.dormir()

# Exibindo os resultados no terminal
print(f"{cachorro_1.nome} agora tem {cachorro_1.comida} unidades de comida.")
print(f"{cachorro_2.nome} está com sono? {cachorro_2.sono}")

File: 2_POO vs Estruturada_alunos/2_comparacao-php_alunos/poo.php
<?php
// ==========================================
// PARTE 2: PROGRAMAÇÃO ORIENTADA A OBJETOS
// ==========================================


// 1ª Digitação (Aqui)


// Exibindo os resultados no navegador
echo "<!DOCTYPE html>
<html lang='pt-BR'>
<head>
    <meta charset='UTF-8'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
    <title>Resultados dos Cachorros (POO)</title>
</head>
<body>
    <h1>Resultados dos Cachorros (POO)</h1>
    <p><strong>{$cachorro_1->getNome()}</strong> agora tem <strong>{$cachorro_1->getComida()}</strong> unidades de comida.</p>
    <p><strong>{$cachorro_2->getNome()}</strong> está com sono? <strong>" . ($cachorro_2->getSono() ? 'Sim' : 'Não') . "</strong></p>
</body>
</html>";
?>